civis_jupyter_notebooks/platform_persistence.py

```diff
@@ -11,6 +11,7 @@
 import sys
 import subprocess
 import requests
+from io import open
```

Comment:
Why do you need this line? Isn't `open` built into python?

Reply:
`io.open` is what's suggested when you want to specify an encoding and have compatibility between Python 2 and 3.  Or do we not need to concern ourselves with that?

http://python-future.org/compatible_idioms.html#file-io-with-open